application.py

```diff
@@ -52,4 +52,10 @@ def register():
             db.session.commit()
             responseMessage = "You registered sucessfully!"
         return render_template("index.html",data=responseMessage)
-    return render_template("register.html")
\ No newline at end of file
+    return render_template("register.html")
+
+@app.route("/admin")
+def listUsers():
+    queryResultSet = Users.query.all()
```

Comment:
Does this display the list in the descending order of the user creation time?

Reply:
No. I changed it to : 
`queryResultSet = Users.query.order_by(desc(Users.registerTime)).all()`